### DISCUSSION TITLE
HIP-338: Signature and Wallet Providers

### DISCUSSION THREAD

**Discussion by sergmetelin** [hiero-ledger/hiero-improvement-proposals#355] [Edits]
This discussion is to discuss HIP-338 - Signature and Wallet Providers 

https://github.com/hashgraph/hedera-improvement-proposal/blob/master/HIP/hip-338.md

@danielakhterov

**Comment by derek-meeco** [comment]
Feb Update:
https://twitter.com/SergMetelin/status/1491753643367878659

**Comment by derek-meeco** [comment]
Feature Pull request merged
https://github.com/hashgraph/hedera-sdk-js/pull/960
